packages/fast-viewer-react/webpack.config.js

```diff
@@ -1,50 +1,54 @@
-const webpack = require("webpack");
 const path = require("path");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 
 const appDir = path.resolve(__dirname, "./app");
 const outDir = path.resolve(__dirname, "./www");
 
-module.exports = {
-    devtool: "inline-source-map",
-    entry: path.resolve(appDir, "index.tsx"),
-    output: {
-        path: path.resolve("./www"),
-        publicPath: "/",
-        filename: "app.js"
-    },
-    mode: process.env.NODE_ENV || "development",
-    resolve: {
-        extensions: [".js", ".ts", ".tsx"],
-    },
-    module: {
-        rules: [
-            {
-                test: /\.tsx?$/,
-                use: [
-                    {
-                        loader: "ts-loader",
-                        options: {
-                            compilerOptions: {
-                                declaration: false,
+module.exports = (env, args) => {
+
+    const isProduction = args.mode === "production";
+
+    return {
+        devServer: {
+            compress: false,
+            historyApiFallback: true,
+            open: true,
+            port: 3000
+        },
+        devtool: isProduction ? "none" : "inline-source-map",
+        entry: path.resolve(appDir, "index.tsx"),
+        mode: args.mode || "development",
+        module: {
+            rules: [
+                {
+                    test: /\.tsx?$/,
+                    use: [
+                        {
+                            loader: "ts-loader",
+                            options: {
+                                compilerOptions: {
+                                    declaration: false,
+                                }
                             }
                         }
-                    }
-                ],
-                exclude: /node_modules/
-            }
-        ]
-    },
-    plugins: [
-        new HtmlWebpackPlugin({
-            title: "FAST viewer",
-            contentBase: outDir
```

Comment:
Does all of this really pertain to a security vulnerability? This seems out of scope